### ISSUE TITLE
AWS::Athena::WorkGroup and AWS::Athena::NamedQuery -> Not found

### ISSUE THREAD

**Issue by oijkn** [JamesWoolfenden/sato#15]
**Describe the bug**
When I run the command `sato parse -f athena.yaml`, the following warnings appear in the log:
```sh
10:07AM WRN AWS::Athena::WorkGroup not found
10:07AM WRN AWS::Athena::NamedQuery not found
```

As a result, the Terraform configuration generation fails.

**To Reproduce**

1. Create a file named athena.yaml with the following configuration:
```yml
Resources:

  athenworkgroup:
    Type: AWS::Athena::WorkGroup
    Properties:
      Name: "my-workgroup"
      State: "ENABLED"

  securityhubAthenaView:
    Type: AWS::Athena::NamedQuery
    Properties:
      Name: AWS-athena
      Database: !Ref database
      WorkGroup: !Ref athenworkgroup
      QueryString: >
          SELECT * FROM "my_table" WHERE "severity" = 'CRITICAL'
```
2. Execute the command: `sato parse -f athena.yaml`.
3. Observe the warnings in the log.

**Expected behavior**
The configuration should be parsed without warnings, and the Terraform configuration should be generated successfully.

**Screenshots**
N/A

**Additional context**
Any guidance or fixes for this issue would be greatly appreciated. Thank you!

**Comment by JamesWoolfenden** [comment]
thanks for logging this ill take a look

**Comment by JamesWoolfenden** [comment]
resolved in 0.0.17
